### PR TITLE
Fixes #12763: Document "JSON select" operator for node Properties

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/12763

### CODE REVIEW

File: 21_node_management/22_search_nodes.txt [hunk 1]
```diff
@@ -211,29 +150,135 @@ The name of the fields tells you about what is expected. It would look like
...
+//select because "city" exists (whatever its value)
+'Properties', 'Name=Value', +Name:JSON Path+, 'datacenter:$.location.address.city'
+
+//select based on a regex on a specific JSON leaf
+'Properties', 'Name=Value', +Name:JSON Path+, 'datacenter:$.location[?(@.name=~/Paris-[0-9]+/)]'
+
+//select based on belonging to a set
+'Properties', 'Name=Value', +Name:JSON Path+, 'datacenter:$.[?(@.environements subsetof ['prod', 'qa', 'dev'])]'
+
+====
```
Comment (amousset) [inline 21_node_management/22_search_nodes.txt]: missing empty line

File: 20_usage/20_quick_search_anything.txt [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+You might have noticed the small text area at the top of the Rudder interface:
+it is the Quick Search bar. Its purpose is to enable a user to easily search for
+Rudder elements (Nodes, Groups, Directives, Parameters, Rules)  based on their
+name, id, description, inventory...
+
+image::./images/rudder-quicksearch.png[Quicksearch]
+
+An autocompletion list will appear as soon as Rudder detects an element it can
+identify, you just have to click on it to be redirected to the element
```
Comment (amousset) [inline 20_usage/20_quick_search_anything.txt]: element's

File: 21_node_management/22_search_nodes.txt [hunk 3]
```diff
@@ -6,177 +6,116 @@
...
+- 2. Select the matching rule
 
-.in: keywords (parameters)
-[frame="topbot",options="header"]
-|======================
-|keyword |Search for
-|parameter_name |Parameter names
-|parameter_value |Parameter values
-|======================
+// FIXME: due to a bug in asciidoc, we cannot write '≠'
```
Comment (amousset) [inline 21_node_management/22_search_nodes.txt]: is it still true ?

File: 21_node_management/22_search_nodes.txt [hunk 4]
```diff
@@ -211,29 +150,135 @@ The name of the fields tells you about what is expected. It would look like
...
+
+* 'Properties', 'Name=Value', +Name:JSON Path+, 'datacenter:json-path-query'.
+
+====
+
+Where +json-path-query+ is JSON path selector from https://github.com/json-path/JsonPath.
+Nodes are selected if the JSON path selector result is not empty, i.e if the JSON value
+contains selected elements. 
+
+For example, let say you have a node with a property whose name is +datacenter+ and value is:
```
Comment (amousset) [inline 21_node_management/22_search_nodes.txt]: let's

File: 21_node_management/22_search_nodes.txt (hunk 3 above)
Comment (fanf) [reply 21_node_management/22_search_nodes.txt]: no idea :) I test and we will see if it breaks

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
PR rebased

**Pull request merged**